packages/store/addon/-private/system/promise-proxies.js

```diff
@@ -17,7 +17,7 @@ import { Promise } from 'rsvp';
   updated when the promise resolves.
 
   For more information see the [Ember.PromiseProxyMixin
-  documentation](/api/classes/Ember.PromiseProxyMixin.html).
+  documentation](/api/classes/PromiseProxyMixin.html).
```

Comment:
I think we should use:

```suggestion
  documentation](/ember/release/classes/PromiseProxyMixin).
```

Specifically, using `/release/` there will always point to the current Ember release (and avoids the issue that @hjdivad was pointing out).